source/jobs.c

```diff
@@ -246,6 +253,8 @@ static void writePreamble( char * buffer,
 #define checkCommonParams() \
     ( ( buffer != NULL ) && ( length > 0U ) && checkThingParams() )
 
+/** @endcond */
+
 /**
  * See jobs.h for docs.
```

Comment:
Is this duplicated documentation of the public-facing API?

Reply:
There public-facing API is documented in the .h file.